lib/BrowserFetcher.js

```diff
@@ -287,6 +288,8 @@ function httpRequest(url, method, response) {
     else
       response(res);
   };
+  if (opt.ca && options.protocol === 'https:')
+    options.ca = opt.ca;
```

Comment:
I don't see node.js API to accept a "ca" option here: https://nodejs.org/api/https.html#https_https_request_options_callback

Is this a thing?

Reply:
Well, Node API states : 
> The following additional options from tls.connect() are also accepted: ca, cert, ciphers, clientCertEngine, crl, dhparam, ecdhCurve, honorCipherOrder, key, passphrase, pfx, rejectUnauthorized, secureOptions, secureProtocol, servername, sessionIdContext.